changelogs/unreleased/ghs-121-too-big-buffer-size.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/app
+
+* Fixed integer overflow issue net.box (ghs-121).
```

Comment:
- **in** net.box.
- We never used the "app" section in changelogs. I'm not sure what section we should use for net.box, but I'd use "bugfix/core".

Reply:
Fixed.